src/agent/onefuzz-agent/src/tasks/config.rs

```diff
@@ -26,11 +26,13 @@ pub struct CommonConfig {
 
     pub instance_id: Uuid,
 
-    pub instrumentation_key: Option<Uuid>,
-
     pub heartbeat_queue: Option<Url>,
 
-    pub telemetry_key: Option<Uuid>,
+    #[serde(alias = "instrumentation_key")]
```

Comment:
For posterity, please document the reason for these aliases, and when we can remove them.